src/apps/src/main/scala/osmesa/apps/batch/ChangesetStatsCreator.scala

```diff
@@ -1,21 +1,21 @@
-package osmesa.analytics.oneoffs
+package osmesa.apps.batch
 
 import java.net.URI
-import java.sql._
 
 import cats.implicits._
 import com.monovore.decline.{CommandApp, Opts}
 import org.apache.spark.TaskContext
 import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.functions._
 import osmesa.analytics.Analytics
+import osmesa.analytics.oneoffs.MergeChangesetUtils
```

Comment:
Probably not a good import to use generally.  If there's useful functionality here, I recommend moving it to a less application-specific location (say to a utils pacakge object?).  (But is this import even used?)

Reply:
This is a rider object on MergeChangesets.scala which got moved to osmesa.apps.batch in the next commit I haven't yet pushed. So this Utils object was moved to osmesa.apps. 

I'll see about putting it somewhere more useful.